Sources/Card/CardClient.swift

```diff
@@ -23,23 +26,97 @@ public class CardClient {
 
     /// Approve an order with a card, which validates buyer's card, and if valid, attaches the card as the payment source to the order.
     /// After the order has been successfully approved, you will need to handle capturing/authorizing the order in your server.
-    /// - Parameter request: The request containing the card and order id for approval
+    /// - Parameters:
+    ///   - orderId: Order id for approval
+    ///   - request: The request containing the card
+    ///   - context: The ASWebAuthenticationPresentationContextProviding protocol conforming ViewController
     /// - Returns: Card result
     /// - Throws: PayPalSDK error if approve order could not complete successfully
-    public func approveOrder(request: CardRequest) async throws -> CardResult {
-        let confirmPaymentRequest = try ConfirmPaymentSourceRequest(
-            card: request.card,
-            orderID: request.orderID,
-            clientID: config.clientID
-        )
-        let (result, _) = try await apiClient.fetch(endpoint: confirmPaymentRequest)
+    public func approveOrder(
+        orderId: String,
```

Comment:
We should revert `orderID` back to being a property on `CardRequest` to line up with the Android API. The goal is to keep the parameter count on `approveOrder` low.